src/components/AddAComment.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect } from "react";
+import { CommentOutlined } from "@ant-design/icons";
+import { FloatButton } from "antd";
+import { postComment } from "../../api";
+import { useParams } from "react-router-dom";
+
+const AddNewComment = (props) => {
+  const { article_id } = useParams();
+  const { comments, setComments } = props;
+  const [commentBody, setCommentBody] = useState();
+  const [newComment, setNewComment] = useState({
+    username: "grumpy19",
+    body: "",
+  });
+
+  const focusOnCommentBox = () => {
+    document.getElementById("add-comment").focus();
+  };
+
+  const submitHandler = (event) => {
+    event.preventDefault();
+    setNewComment({ ...newComment }, (newComment.body = commentBody));
+
+    postComment(article_id, newComment).then((newCommentFromAPI) => {
+      if (newCommentFromAPI.code) {
+        
```

Comment:
not sure whats supposed to happen here?

Reply:
Whoops honestly no idea, think that was from me trying to do error handling then I decided to do it during task 12 instead - deleted!